### PR TITLE
Previous points addition and bug fixes

### PR DESCRIPTION
**Pull request by timaeus321**

### CODE REVIEW

File: app/views/previous_points/accomplishments.html.erb [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+<table class="table" summary="Member Accomplishments List">
+
+    <tr class="header">
+        <th>Name</th>
+		<th>Description</th>
+		<th>Points</th>
+		<th>When</th>
+    </tr>
+    <% @member.accomplishments.where(:semester_id == @sem_id).each do |accomplishments| %>
```
Comment (pulchroxloom) [inline app/views/previous_points/accomplishments.html.erb:23]: Does this work? I thought you were supposed to do:
```rb
where(semester_id: @sem_id).each
```

File: app/views/previous_points/events.html.erb [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+	<tr class="header">
+		<th>Name</th>
+		<th>Points</th>
+		<th>Date</th>
+		<th>Type</th>
+		<th>Details</th>
+	</tr>
+	<% @curr_semester.each do |sem| %>
+	<% @member.events.where(sem.dates === :start_time).each do |event| %>
```
Comment (pulchroxloom) [inline app/views/previous_points/events.html.erb:27]: Does this work? I thought you were supposed to do:
```rb
where(semester_id: @sem_id).each
```
In fact, I didn't know that `===` was a valid Ruby operator.

File: app/views/previous_points/manual_points.html.erb [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+<table class="table" summary="Manual Points List">
+
+    <tr class="header">
+    <th>Points</th>
+		<th>Reason</th>
+		<th>Category</th>
+		<th>When</th>
+    </tr>
+    <% @member.manual_points.where(:semester_id == @sem_id).each do |manual_points| %>
```
Comment (pulchroxloom) [inline app/views/previous_points/manual_points.html.erb:25]: Does this work? I thought you were supposed to do:
```rb
where(semester_id: @sem_id).each
```

File: app/views/previous_points/events.html.erb (hunk 2 above)
Comment (pulchroxloom) [reply app/views/previous_points/events.html.erb:27]: Wait, this actually won't work because of how we changed the way `Semester.current_semester` works.

**Review by pulchroxloom — CHANGES_REQUESTED** [review]
I think this needs some changes.

File: app/views/previous_points/accomplishments.html.erb (hunk 1 above)
Comment (dvansteen) [reply app/views/previous_points/accomplishments.html.erb:23]: It should probably be in the controller as it is

File: app/views/previous_points/events.html.erb (hunk 2 above)
Comment (dvansteen) [inline app/views/previous_points/events.html.erb]: change to a range check

File: app/views/previous_points/manual_points.html.erb (hunk 3 above)
Comment (dvansteen) [inline app/views/previous_points/manual_points.html.erb:25]: here too

File: app/views/previous_points/events.html.erb (hunk 2 above)
Comment (pulchroxloom) [reply app/views/previous_points/events.html.erb]: Improved this a little bit, so resolving.

**Review by dvansteen — APPROVED** [review]
Looks good

File: app/views/previous_points/events.html.erb (hunk 2 above)
Comment (pulchroxloom) [reply app/views/previous_points/events.html.erb]: Changed to proper ruby syntax. It's going to implicitly do a range check.

File: app/views/previous_points/manual_points.html.erb (hunk 3 above)
Comment (pulchroxloom) [reply app/views/previous_points/manual_points.html.erb]: This is not ranges, but I changed it to the correct syntax.

**Pull request merged**